tests/code_file_manager_test.py

```diff
@@ -66,3 +66,26 @@ def test_partial_files(mock_config):
              4:fourth
              5:fifth
               """)
+
+
+def test_run_from_subdirectory(mock_config, temp_testbed):
+    """Run mentat from a subdirectory of the git root"""
+    # Change to the subdirectory
+    run_from_dir = mock_config.git_root / "multifile_calculator"
```

Comment:
This can just be temp_testbed / "multifile_calculator"; temp_testbed just returns the path to the current temporary file. Additionally, I bet we could just run os.chdir("multifile_calculator") and avoid having to reference the testbed at all; feel free to do whichever you prefer!